standalone/src/screens/ConfirmationScreen.tsx

```diff
@@ -1,27 +1,27 @@
 import Grid from '@material-ui/core/Grid';
-import * as React from 'react';
-
+import React, { useRef } from 'react';
 import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
 
 import '../css/Common.scss';
 import '../css/ConfirmationScreen.scss';
 
-import {
-    CredentialKeyFieldsProps,
-    CredentialKeyFieldState,
-    ConfirmationProps
-} from '../interfaces/ConfirmationInterfaces';
+import { CredentialKeyFieldsProps, CredentialKeyFieldState, ConfirmationProps } from '../interfaces/ConfirmationInterfaces';
 
 import FlowDispatchTypes from '../enums/FlowDispatchTypes';
 
+
 export default function ConfirmationScreen(props: ConfirmationProps) {
+    const consent = (): void => {
+        const type: string = FlowDispatchTypes.NEXT;
+        props.dispatch({ type });
+    };
+
     return (
         <div className="Confirmation screen">
-            <Grid
-                container
+            <Grid container
                 direction="column"
-                justifyContent="center"
+                justify="center"
```

Comment:
It should be `justifyContent` rather than `justify` - the `justify` prop has been deprecated by MaterialUI